plugins/schema/src/types.ts

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright © 2024 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import { type Class, type Obj, type Ref } from '@hcengineering/core'
+import type { JSONSchema7 } from 'json-schema'
+
+/** @public */
+export interface Schema extends Obj, JSONSchema7 {}
```

Comment:
@aplatoff Let's move here, so we could resolve the thread later.

> Just curious: why do not use `platform`'s model for that? Is there any limitations and it can not cover some cases?

What exactly do you mean? The module does use `platform`'s native data structures like `Obj`, `Doc` and `Mixin`. No limitations there, they do perfectly fit our needs 👍🏽

Reply:
I mean that since both the platform and JSON Schema are focused on modeling data structures, why do we need two different mechanisms for modeling?

Reply:
Any comments on this? :) @anotherpit

Reply:
Discussed on call. The suggested idea is to not reinvent the wheel and utilize `Class`es and `Attribute`s and to have a flow similar to that of the Class Settings screen

<img width="1512" alt="image" src="https://github.com/hcengineering/platform/assets/222219/8ad02b0b-54b1-41d4-a7d6-249111cfdb4e">